### DISCUSSION TITLE
Modeling Liquid CO2 droplet evaporation and cooling

### DISCUSSION THREAD

**Discussion by Muthu-V** [firemodels/fds#13572] [Q&A] [answered]
Dear FDS Developers and Users,  I see FDS supports liquid droplet models and solid particle models to exchange heat and mass transfer to the ambient atmosphere. Since FDS don't have liquid CO2 properties by default, It can be entered manually.  When liquid droplets are ejected at certain speed to the ambience  it starts to cooldown due to evaporation  and droplet temperatures are becoming below  the initial liquid droplet temperature .  It may reach  dry ice temperature -80 C in the near filed . 
 Is it possible to treat those particle those are below -80 C as solids and sublimate them into gas phase in the same model  ? 
for scenarios where  joule-thomson effects  are negligible during depressurization, Can FDS be used for evaporative cooling calculations due to depressurization of Liquid CO2 to the atmosphere  ? Are there any caveats or limitations in the model  ?

**Comment by drjfloyd** [comment, marked as answer]
You cannot do this with the droplet model (e.g. SPEC_ID on PART), but you could approximate this as "solid" particles (e.g. SURF_ID on PART).  You could define a solid and a liquid CO2 material. The solid material would have two reactions: melting to liquid and sublimation to gas. The liquid would have two reactions, freezing to solid and evaporation to gas.

Look at the ice_cube.fds verfication case. This has a solid block of ice that melts to create drops.  In something similar to your case if you wanted small hail to melt you would have insteand the OBST a PART with SURF_ID where the SURF was consisted of the ice MATL. On the ice MATL you would have a MATL_ID for a liquid water MATL instead of the PART_ID for water droplets.

**Reply by Muthu-V** [reply]
Thank you for your inputs and reference to validation case. It was useful.
